### ISSUE TITLE
Any reason you didn't port react-chartjs-2

### ISSUE THREAD

**Issue by ZanzyTHEbar** [s0ftik3/solid-chartjs#3]
Awesome library. I started porting `react-chartjs-2` and then found your library - haha. Should've looked first. 

Will test this and let you know how it goes. 

I did get 98% of the porting done though, before i found yours. Take a look [here](https://github.com/ZanzyTHEbar/solidjs-chartjs)

**Comment by s0ftik3** [comment]
Hey! Thanks for letting me know. I didn't think about porting it from React because don't use it much, so I wasn't aware of the already existing solution. I've created this one mostly for work purposes, rather then for public use, just I thought it might be useful for people and published. However, I've checked the React one and I think I could redo some parts of my code and improve the library itself. Feel free to contribute if you have time and wish. Thanks again, I'll wait for your feedback
